packages/eds-core-react/src/components/SideBar/SideBar.tsx

```diff
@@ -46,7 +46,7 @@ const GridContainer = styled.div<ContainerProps>(({ theme, open }) => {
       'content'
       'footer';
     background-color: ${theme.background};
-    overflow: hidden;
+    overflow: scroll;
```

Comment:
This needs to be `overflow-x: auto;` or Sidebar will look like this for 90% of our users:  
![bilde](https://user-images.githubusercontent.com/4499531/227536403-6297b99b-2e7b-4714-a412-d4a3b76617b7.png)
Additionally something is causing overflow in the Y direction, this should also be identified and fixed